docs/alerts.rst

```diff
@@ -68,6 +68,32 @@ You can also filter on different kind of **TTPs** (*Tools Techniques Procedures*
     ...
 
 
+Retrieving Alerts for Multiple Organizations
+--------------------------------------------
+
+With the example below, you can retrieve alerts for multiple organizations.
+
+Create a csv file with values that match the profile names in your credentials.cbc file. You can modify the code inside the for loop to fit your needs.
+
+.. code-block:: python
+
+ >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import BaseAlert
+    >>> import csv
+    >>> file = open ("data.csv", "r", encoding='utf-8-sig')
+    >>> org_list = list(csv.reader(file, delimiter=","))
```

Comment:
Why use a csv file instead of creating a list of profile names in the code itself?

Reply:
This is just an example, maybe this example actually _looks_ more like a use case one of our customers would have?

Reply:
If we are going to use the csv file method then would recommending opening the file using `with:` as this guarantees close if something goes wrong

Reply:
@abowersox-cb you'e correct. I was going for simplicity, because the audience for this script is not very familiar with python. @avanbrunt-cb I'll look into using with:

Reply:
Done